VotingApplication/VotingApplication.Web/Scripts/Controllers/Create/CreateController.js

```diff
@@ -68,7 +60,8 @@
                 MaxPerVote: null,
                 MaxPoints: null,
                 Invitations: [],
-                OptionAdding : false
+                OptionAdding: false,
+                DoesExpire: false
```

Comment:
`ExpiryDate` is referenced in `ConfirmStepController.js` on `$scope.newPoll`, but isn't added here.